crates/storage/src/counter.rs

```diff
@@ -96,3 +96,53 @@ where
         Ok((old_value, new_value))
     }
 }
+
```

Comment:
Add a "separator" comment as in all other files

```suggestion

// ----------------------------------- tests -----------------------------------

```